app/src/main/java/edu/grinnell/grinnell_publications_android/Views/Activities/Settings.java

```diff
@@ -0,0 +1,18 @@
+package edu.grinnell.grinnell_publications_android.Views.Activities;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import edu.grinnell.grinnell_publications_android.R;
+
+/*
+    An activity to display relevant settings and configuration options
+ */
+public class Settings extends AppCompatActivity {
```

Comment:
More like MainSettings -- we might have other settings in the future. More specific = better.